packages/frontend/core/src/modules/app-sidebar/views/index.tsx

```diff
@@ -147,13 +151,39 @@ export function AppSidebar({ children }: PropsWithChildren) {
     };
   }, [appSidebarService, resizing, sidebarState, width]);
 
+  const resizeHandleDropTargetOptions = useMemo(() => {
+    return () => ({
+      data: () => {
+        const firstView = workbenchService.views$.value.at(0);
+
+        if (!firstView) {
```

Comment:
firstView always exists?

Reply:
I think so but here is to get rid of the type error (`at` returns `T | undefined`)